DS2S META/Utils/Offsets/DS2VOffsets.cs

```diff
@@ -22,7 +22,7 @@ public DS2VOffsets()
             PlayerName = new(0xA4);
             ForceQuit = new(0xDF1);
             PlayerCtrl = new(HP: 0xFC, HPMin: 0x100, HPMax: 0x104, HPCap: 0x108, 
-                             SP: 0x140, SPMax: 0x148, SpeedModifier: 0x208, CurrPoise:0x1ac);
+                             SP: 0x140, SPMax: 0x148, SpeedModifier: 0x208, CurrPoise:0x1ac, SPMin: 0x144); // Need to find the value for vanilla, testing for scholar for now
```

Comment:
Will leaving this value in here cause any issues for vanilla?

Reply:
Also another comment I forgot to remove sorry, it is currently working for vanilla so this comment is not needed anymore